### ISSUE TITLE
weave scope service for systemd

### ISSUE THREAD

**Issue by thatsk** [weaveworks/scope#2615]
[Unit]
Description=Scope UI
Documentation=https://www.weave.works/docs/scope/latest/introducing/
Requires=weave.service
After=weave.service

[Service]
EnvironmentFile=-/etc/sysconfig/scope
ExecStart=/usr/bin/scope launch $HOST
ExecStop=/usr/bin/scope stop

[Install]
WantedBy=multi-user.target

Unfortunately this service file is not working. But when i pass through command line it works. But when i try in systemd file it gives an error in logs.
: Unregistered Authentication Agent for unix-process:7103:179995403 (system bus name :1.727
Jun 19 09:41:26 test1.com  kernel: overlayfs: upper fs needs to support d_type. This is an invalid configuration.

why its not pass through systemd file

**Comment by 2opremio** [comment]
CC @errordeveloper

**Comment by thatsk** [comment]
?

**Comment by 2opremio** [comment]
@thatsk Where did you get that systemd file? Did you create it yourself?

**Comment by thatsk** [comment]
Yes . Because its tedius to launch the command everytime . and we don't know where is kill so i have also to check the logs. systemd is best option for service ui Similar to weave net

**Comment by 2opremio** [comment]
Understood. Let's see if @errordeveloper can help you. However, we don't officially support launching through Systemd.

**Comment by calvintam236** [comment]
It would be great to let scope launch itself on boot (as a service)...

**Comment by thatsk** [comment]
is there any update?

**Comment by calvintam236** [comment]
Meanwhile, I think there is a workaround. It looks like the scope script basically runs the docker container. Docker has a [restart policy](https://docs.docker.com/engine/admin/start-containers-automatically/) that can make it restart after certain events.

Can you change the scope script download from git.io/scope?

**Comment by thatsk** [comment]
[scope.txt](https://github.com/weaveworks/scope/files/1147417/scope.txt)
Added please check i have configured as per weave net policy that restart always

**Comment by calvintam236** [comment]
maybe @2opremio can make those changes by @thatsk into the source code for everyone? that would be awesome

**Comment by rade** [comment]
You can already set the restart policy, or supply any other additional docker arguments, by starting scope with
```sh
WEAVESCOPE_DOCKER_ARGS="--restart=always" scope launch
```
except at the moment this breaks because `$WEAVESCOPE_DOCKER_ARGS` is also passed to the arg check run, which has a `--rm` and hence fails with
```
docker: Conflicting options: --restart and --rm.
```

I'm be tempted to remove the `$WEAVESCOPE_DOCKER_ARGS` from that call. @2opremio, what do you reckon?

**Comment by 2opremio** [comment]
> I'm be tempted to remove the $WEAVESCOPE_DOCKER_ARGS from that call. @2opremio, what do you reckon?

Makes sense. We could check whether `--restart` is passed too.

**Comment by rade** [comment]
> I'm be tempted to remove the `$WEAVESCOPE_DOCKER_ARGS` from that call.

I did that in #2715, which was released in 1.6.0.

**Comment by thatsk** [comment]
@rade @2opremio  is it done or we have to update. Also please update the doc because user is interested in this
